grommet-leaflet-core/src/markers/Cluster.jsx

```diff
@@ -54,11 +49,19 @@ const Cluster = ({ cluster, ...rest }) => {
   const count = cluster.getChildCount();
   const status = getClusterStatus(cluster.getAllChildMarkers());
 
-  const border = {
-    color: STATUS_MAP[status].color,
-    size: STATUS_MAP[status].borderSize || 'small',
+  let border = {
+    color: '#000000B8',
```

Comment:
Can this be a theme namespace instead?

Reply:
Weirdly the theme is coming through as undefined here... Still trying to understand why that is happening

Reply:
I think the reason this is happening is because `getClusterStatus` is being called outside of the grommet wrapper. I think we'll need to pass `theme` as a parameter to the function.

Reply:
Oops I misread the diff, but my first hunch is wondering if it's because we're outside of the grommet wrapper. As I'm looking at the details it seems like the the hpe theme text colors also aren't coming through anymore

Reply:
Yeah it looks like the base theme colors are being used for the status markers

Reply:
Leaving this for now, I think we can clean up the theme context behavior in a separate PR